### ISSUE TITLE
Support for Plan 9

### ISSUE THREAD

**Issue by fhs** [rjkroege/edwood#115]
To get edwood working in Plan 9, there are some syscall usage that needs to be removed and `9fans.net/go/draw` needs to be ported.

I've been slowly working on this.

**Comment by fhs** [comment]
After #125, Edwood is working pretty well in Plan 9. You need to be using my [plan9 branch](https://github.com/fhs/go/tree/plan9) of 9fans.net/go package. I need to clean up the changes (and remove hacks) before I'll send it upstream.

**Comment by rjkroege** [comment]
Very cool! Does it run on HarveyOS too?

**Comment by sirnewton01** [comment]
I can give it a try on 9front if nobody else has.

What's a good way to test the different platforms for regressions? We've tried just making sure it builds with travis on GOOS=plan9 on driusan/dgit and sirnewton01/ghfs but that only checks that the OS-specific modules compile, not that they actually work.

**Comment by rjkroege** [comment]
@sirnewton01: I heard that the Go team has a 9front GCE image that they use for running the Go regression tests on Plan9. I'll file a separate issue here for testing on Plan9.

Also: [sirnewton01/ghfs](https://github.com/sirnewton01/ghfs) is very cool. I'll ask you to have markdown previews via #70 when I get it done. 😊

**Comment by fhs** [comment]
If anyone want to try out edwood on plan9, this is how you can build it:
```
go mod edit -replace 9fans.net/go=github.com/fhs/9fans-go@v0.0.0-fhs.20210824
go mod download 9fans.net/go
GOOS=plan9 go build
./edwood
```

**Comment by rjkroege** [comment]
@fhs I should adjust the mod file to point at your branch?

**Comment by fhs** [comment]
> @fhs I should adjust the mod file to point at your branch?

@rjkroege I'm not sure `go install` will be happy with the replace directive. We can give it a try, but I'd like to see a more official fork of `9fans.net/go` (maybe by Harvey devs?).
